tests/runtime/call

```diff
@@ -232,27 +232,44 @@ ENV BPFTRACE_STACK_MODE=bpftrace
 EXPECT uprobeFunction1+0
 TIMEOUT 5
 AFTER ./testprogs/uprobe_loop
+# This was debugged as far down as std::cout having badbit [0] set after a
+# write. It's really confusing why. Cannot be reproduced locally. While
+# debugging is ongoing disable in CI. There's a good chance we never figure out
+# why. My current best guess is that somewhere in all the layers of shelling
+# out there's a misbehaving pipe that hangs up too early.
+#
+# Note that CI is a default environment variable set by GHA [1].
+#
+# [0]: https://en.cppreference.com/w/cpp/io/ios_base/iostate
+# [1]: https://docs.github.com/en/actions/learn-github-actions/variables#default-environment-variables
+SKIP_IF_ENV_HAS CI=true
 
 NAME ustack_stack_mode_env_perf
 PROG u:./testprogs/uprobe_loop:uprobeFunction1 { printf("%s\n", ustack(1)); exit(); }
 ENV BPFTRACE_STACK_MODE=perf
 EXPECT_REGEX \d+ uprobeFunction1\+0 \(.*/uprobe_loop\)
 TIMEOUT 5
 AFTER ./testprogs/uprobe_loop
+# See above
```

Comment:
Can we add a link to an existing issue here instead of `#See above` ?

Reply:
Yeah good idea